app/components/scrollingCalendar/helpers.ts

```diff
@@ -976,9 +976,10 @@ export const updateOrInsertWorkWeekInProject = (
             } else {
               updatedWorkWeeks = [...assignment.workWeeks, workWeek];
             }
+            const updatedFields = {workWeeks: updatedWorkWeeks, canBeDeleted:false};
```

Comment:
I think this logic not working currently, notice two things: 
1. When entered estimated hours to the project/assignment - delete btn disappeared - it should disappear only if we entered actual hours I think. 
2. When delete all actual hours from project - it doesn`t appears, but should?

Reply:
1. I agree
2. I believe the original intent of canBeDeleted is based on if you have workWeeks and at least in my database the workWeeks don't delete but the hours are set to zero. What is your opinion here?

Reply:
> I am also noticing that when someone just erases their actualHours and submits the workWeek input empty it does not change the actualHours, is that the intended result?

good catch, I think it`s a bag. Want to create an issue? What do you about the behavior in this case? we should send 0 to delete hours?

Reply:
yes I think sending zero makes the most sense in this case

Reply:
> 1. I agree
> 2. I believe the original intent of canBeDeleted is based on if you have workWeeks and at least in my database the workWeeks don't delete but the hours are set to zero. What is your opinion here?

As for canBeDeleted. I think for clear UX if assignment have no actual hours user should be able to delete this assignment as expectable behaviour. We can have work weeks only with estimated hours, but actual hours can be set to zero, Do we need additional logic for checking if some workweeks have actual hours more then 0?